### PR TITLE
backport: trivial 2024 01 13

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
## Issue being fixed or feature implemented
Batch of trivial backports

## What was done?
trivial backports

## How Has This Been Tested?
CI

## Breaking Changes
None

## Checklist:
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

**Review by UdjinM6 — APPROVED** [review]
utACK

File: contrib/verify-commits/trusted-git-root [hunk 1]
```diff
@@ -1 +1 @@
-82bcf405f6db1d55b684a1f63a4aabad376cdad7
+577bd51a4b8de066466a445192c1c653872657e2
```
Comment (knst) [inline contrib/verify-commits/trusted-git-root:1]: > Merge https://github.com/bitcoin/bitcoin/pull/25197: contrib: Remove keys that are no longer used for…

we need to dashify it.

```suggestion
3b7deea3b51a5c6b466a78bfbfcdfed48772a2f5
```

File: contrib/verify-commits/trusted-keys [hunk 2]
```diff
@@ -1,7 +1,5 @@
 71A3B16735405025D447E8F274810B012346C9A6
 133EAC179436F14A5CF1B794860FEB804E669320
-32EE5C4C3FA15CCADB46ABE529D4BCB6416F53EC
 B8B3F1C0E58C15DB6A81D30C3648A882F4316B9B
-CA03882CB1FC067B5D3ACFE4D300116E1C875A3D
 E777299FC265DD04793070EB944D35F9AC3DB76A
 D1DBF2C4B96F2DEBF4C16654410108112E7EA81F
```
Comment (knst) [inline contrib/verify-commits/trusted-keys:5]: > Merge https://github.com/bitcoin/bitcoin/pull/25197: contrib: Remove keys that are no longer used for…

we need to dashify it.

```suggestion
15191D05B5CF956FE37C95962176C4A5D01EA524
29590362EC878A81FD3C202B52527BEDABE87984
3F5D48C9F00293CD365A3A9883592BD1400D58D9
FD4A3062EE42C95FE9B34DBC6317F01E6F491072
```
(sorted alphabetically)

**Review by knst — CHANGES_REQUESTED** [review]
even after dashification there's still something wrong with bitcoin-25197

I tried to use different trusted-git-root, but `contrib/verify-commits/verify-commits.py` still fails.... 

I am not sure how to fix it without digging, maybe better to drop 25197?

File: contrib/verify-commits/trusted-git-root (hunk 1 above)
Comment (PastaPastaPasta) [reply contrib/verify-commits/trusted-git-root:1]: Nah; these changes should be in new PRs; we don't actually use these scripts sooo... 🤷 feel free to open other PRs

**Review by knst — APPROVED** [review]
> these changes should be in new PRs; 

Ok, utACK then

**Pull request merged**